mlflow/tracking/client.py

```diff
@@ -1422,108 +1424,12 @@ def _is_plotly_figure(fig):
     def log_image(
         self,
         run_id: str,
-        image: Union["numpy.ndarray", "PIL.Image.Image"],
+        image: Union["numpy.ndarray", "PIL.Image.Image", "Image"],
```

Comment:
```suggestion
        image: Union["numpy.ndarray", "PIL.Image.Image", "mlflow.tracking.multimedia.Image"],
```
Perhaps Sphinx needs the explicit path in order to resolve the typing correctly?